yarn.lock

```diff
@@ -2,29 +2,50 @@
 # yarn lockfile v1
 
 
-"@babel/code-frame@7.0.0", "@babel/code-frame@^7.0.0":
+"@babel/code-frame@7.8.3":
```

Comment:
I didn't check this file, but I think we can assume it is correct if it passed the CI checks and if it will pass a review